modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-storage-class_{context}"]
+= Installing the OpenStack Manila CSI Driver Operator
+
+The CSI Manila Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the CSI Manila driver in your cluster.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* NFS share types are used for provisioning.
+//This prereq won't be required unless there are custom install instructions that are added to OCP docs:
+//* The OpenStack Manila service has been correctly set up and is accessible from the {product-title} cluster.
+
+.Procedure
+
+To create the `manila-csi` project from the web console, follow these steps:
```

Comment:
You refer to the project as `openshift-csi-driver-manila-operator` below. IMO, `openshift-csi-driver-manila-operator` is the correct one.

Reply:
That was my thinking, too, but was matching what was in the deployment GDoc. Fixed.